base/tks/src/main/java/org/dogtagpki/server/tks/servlet/TokenServlet.java

```diff
@@ -2899,10 +2985,15 @@ private void processComputeSessionKeysSCP03(HttpServletRequest req, HttpServletR
                 sb.append(kekSessionKeyString);
                 sb.append("&" + IRemoteRequest.TKS_RESPONSE_KEK_DesKey + "=");
                 sb.append(kek_wrapped_desKeyString);
+                sb.append("&" + IRemoteRequest.TKS_RESPONSE_KEK_AesKey + "=");
+                sb.append(kek_wrapped_aesKeyString);
                 sb.append("&" + IRemoteRequest.TKS_RESPONSE_KeyCheck + "=");
                 sb.append(keycheck_s);
```

Comment:
note again that I didn't observe call to assign computeKeyCheck_SCP03 to keycheck_s so this is an 8 byte check instead of a 16 byte check.